### PR TITLE
* Updated the font sizes in line with design on front page*

### PR DESCRIPTION
**Pull request by nikhilgupta2001**
Updated the font sizes in line with design on front page

### CODE REVIEW

**Comment by nikhilgupta2001** [comment]
this pr is for #150

File: pages/index.js [hunk 1]
```diff
@@ -155,7 +155,7 @@ class IndexPage extends Component {
           </section>
           <section className='h-section pt0-ns pt4-l'>
             <div className='flex flex-column pt4-ns'>
-              <h2 className='f2-l f3 tf-oswald tc mb5-l mv4'>
+              <h2 className='f2-l f3 tf-oswald tc mb5-l mv4' style={{fontSize:80,fontWeight:"bold"}}>
```
Comment (joelwass) [inline pages/index.js:158]: the font size 40 for the other headers look good, not sure why this is font size 80 and bold

File: package.json [hunk 2]
```diff
@@ -53,5 +53,23 @@
   },
   "engines": {
     "node": "12.x"
-  }
+  },
+  "description": "<div>   <img alt=\"TeacherFund logo\" src=\"https://github.com/teacherfund/TeacherFund_next/raw/production/public/images/Logo.png\" width=\"200px\"> </div>",
+  "version": "1.0.0",
```
Comment (joelwass) [inline package.json:58]: this all needs to be removed

File: components/team.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ const Team = (props) =>
   (
     <section className='h-section mb0-ns pv5-l bg-card'>
       <div className='flex flex-column z-100 pt4-m'>
-        <h2 className='f2-l f3 mv3-m tf-oswald center'>
+        <h2 className='f2-l f3 mv3-m tf-oswald center' style={{fontSize:60 }} >
```
Comment (joelwass) [inline components/team.js:29]: why not be consistent with the fontSize: 40 ?

File: components/nav.js [hunk 4]
```diff
@@ -59,7 +63,7 @@ const Nav = () => {
             {buttons.map(({ key, href, label }) => (
               <div key={key} className='db center w-auto fr ph2'>
                 <Link href={href} key={key}>
-                  <a className='btn-alt no-underline black pv3 ph4 br3'>{label}</a>
+                  <a className='btn-alt no-underline black pv3 ph4 br3' style={{fontSize:25 }}>{label}</a>
```
Comment (joelwass) [inline components/nav.js:66]: i think this is far too big (not sure if the design pixel density is off) let's revert this file

File: components/icons/hamburger.js [hunk 5]
```diff
@@ -3,8 +3,8 @@ const Hamburger = () => (
     <svg className='hamburgerIcon' id='hamburger-icon' xmlns='http://www.w3.org/2000/svg' xlink='http://www.w3.org/1999/xlink' viewBox='0 0 24 24' version='1.1' x='0px' y='0px'>
       <title>Icon/Stroke/Hamburger</title>
       <desc>Hamburger Icon.</desc>
-      <g stroke='none' strokeWidth='1' fill='none' fillRule='evenodd'>
-        <rect x='0' y='0' width='24' height='24' />
+      <g stroke='none' strokeWidth='2' fill='none' fillRule='evenodd'>
+        <rect x='0' y='0' width='50' height='50' />
```
Comment (joelwass) [inline components/icons/hamburger.js:7]: what inspired this change?

File: components/ProcessCard.js [hunk 6]
```diff
@@ -2,7 +2,7 @@ import 'react'
 
 export default function ProcessCard ({ title, children, icon }) {
   return (
-    <div className='ma3 ma3-m w-25-m w-25-l'>
+    <div className='ma3 ma3-m w-30-m w-30-l h-100  card' >
```
Comment (joelwass) [inline components/ProcessCard.js:5]: what does the card class do?

**Review by joelwass — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (nikhilgupta2001) [reply package.json:58]: sir, do you want me to remove description and version

File: components/ProcessCard.js (hunk 6 above)
Comment (nikhilgupta2001) [reply components/ProcessCard.js:5]: i was trying to use card class of bootstrap
but its not working

File: package.json (hunk 2 above)
Comment (joelwass) [reply package.json:58]: description and version are fine, thanks!

**Pull request closed without merging**